modules/openapi-generator/src/main/resources/dart2/class.mustache

```diff
@@ -1,11 +1,16 @@
 class {{classname}} {
   {{#vars}}
-  {{#description}}/* {{{description}}} */{{/description}}
+  {{#description}}/// {{{description}}}{{/description}}
```

Comment:
Correct way of documenting fields https://dart.dev/guides/language/effective-dart/documentation#doc-comments